src/Neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -418,5 +418,14 @@ private bool UnblockAccount(ApplicationEngine engine, UInt160 account)
             engine.SnapshotCache.Delete(key);
             return true;
         }
+
+        [ContractMethod(Hardfork.HF_Gorgon, CpuFee = 1 << 15, RequiredCallFlags = CallFlags.ReadStates)]
```

Comment:
3.9 it's Faun, isn't it?
```suggestion
        [ContractMethod(Hardfork.HF_Faun, CpuFee = 1 << 15, RequiredCallFlags = CallFlags.ReadStates)]
```